.ruby-gemset

```diff
@@ -0,0 +1 @@
+enova_landable
```

Comment:
we try and keep this as enovaless as possible due to its opensourceness, can we just make this landable?